src/app/shared/temporary-consts.ts

```diff
@@ -0,0 +1,28 @@
+import { Prefix, Option } from './models/temporary-models';
+
+export const PHONE_NUMBER_REGEX = /^\d{9}$/;
+
+export const prefixes: Prefix[] = [
+  {
+    countryCode: 'pl_PL',
+    prefix: '48',
+  },
+  {
+    countryCode: 'uk_UA',
+    prefix: '380',
+  },
+];
+
+export const languages: Option[] = [
+  { code: 'pl', label: 'Polski' },
```

Comment:
zobacz SiteHeaderComponent

Reply:
to nie jest moje, mam to poprawić?

Reply:
![image](https://user-images.githubusercontent.com/12514322/157894744-6bf0694a-2419-46b3-b299-9c86a06890a8.png)
Wydaje mi się że ten prop 'text' też pwoinien być enumem a nie stringiem

Reply:
tak chyba już bardziej konsekwentnie:
![image](https://user-images.githubusercontent.com/12514322/157895647-d6436000-3708-4760-9e3a-add030f76fe5.png)

Reply:
Wygląda dobrze, nie zdąże zbadać całego kontekstu to zdaje się na Ciebie